config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml

```diff
@@ -239,24 +238,3 @@ presubmits:
       testgrid-tab-name: pr-azurefile-csi-driver-e2e-windows
       description: "Run E2E tests on a Windows cluster for Azure File CSI driver."
       testgrid-num-columns-recent: '30'
-  - name: pull-azurefile-csi-driver-windows-build
-    decorate: true
-    always_run: true
-    path_alias: sigs.k8s.io/azurefile-csi-driver
-    branches:
-    - master
-    labels:
-      preset-service-account: "true"
-    spec:
-      containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200404-7b4af8e-master
-        command:
-        - runner.sh
-        args:
-        - make
```

Comment:
I would like to keep `make azurefile|azuredisk-windows` since it's quite straighforward to check whether windows build pass or not, also this test won't cost any azure resource